assets/prefabs/Pear.prefab

```diff
@@ -0,0 +1,10 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Pear"
+  },
+  "Item": {
+    "icon": "SimpleFarming:Pear",

```

Comment:
This graphic is addressed correctly, but the image has a white background and is not square, making it look pretty weird in-game. Please resize and make the background transparent - you might be using an image tool that doesn't support transparency, since the original actually is transparent to begin with?